### ISSUE TITLE
amp-autocomplete missing role="combobox"

### ISSUE THREAD

**Issue by nainar** [ampproject/amphtml#28926]
https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/amphtml-discuss/nr6mk9K1KWo/WbaFu9r0BgAJ

I can repro it here: https://glitch.com/edit/#!/join/5aa06819-cc57-40d0-9592-f34dc3779593

role=`combobox` should appear on `amp-autocomplete` and not the child `input` https://www.w3.org/TR/wai-aria-1.1/#combobox
